scripts/globals/weaponskills/chant_du_cygne.lua

```diff
@@ -20,14 +20,17 @@ weaponskill_object.onUseWeaponSkill = function(player, target, wsID, tp, primary
     local params = {}
     params.numHits = 3
     params.ftp100 = 2.25 params.ftp200 = 2.25 params.ftp300 = 2.25
-    params.str_wsc = 0.0 params.dex_wsc = 0.6 params.vit_wsc = 0.0 params.agi_wsc = 0.0 params.int_wsc = 0.0 params.mnd_wsc = 0.0 params.chr_wsc = 0.0
+    params.str_wsc = 0.0 params.dex_wsc = 0.6 params.vit_wsc = 0.0
+    params.agi_wsc = 0.0 params.int_wsc = 0.0 params.mnd_wsc = 0.0
+    params.chr_wsc = 0.0
     params.crit100 = 0.1 params.crit200 = 0.3 params.crit300 = 0.5
```

Comment:
While we're in this file, update the crit rate to match testing retail (this has not changed since 2011)

https://www.bg-wiki.com/ffxi/Chant_du_Cygne

Reply:
Adjusted crit params under the Adoulin WS changes, but I can make the current retail values the default if preferred.

Reply:
there is no "prefer". there is only what we know, and what we know is that CDC's crit bonus never changed in both patch notes. Until evidence of the contrary, the crit rate is the same for "Adoulin WS" (which is not actually really "Adoulin WS", it's just a bad name someone gave it)

WSC updated to 80%:
https://forum.square-enix.com/ffxi/threads/42615

fTP mirror (also reduced base fTP):
https://forum.square-enix.com/ffxi/threads/45365-Dec-10-2014-%28JST%29-Version-Update

Reply:
> (which is not actually really "Adoulin WS", it's just a bad name someone gave it)

I haven't gotten around to it, but I will be updating that to indicate modern rather than incorrectly naming an expansion for it.

Reply:
Please fix crit rate on the non "Adoulin" section and this can get merged